source/appModules/soffice.py

```diff
@@ -199,7 +199,7 @@ class SymphonyIATableCell(SymphonyTableCell):
 
 	def event_selectionAdd(self):
 		curFocus = api.getFocusObject()
-		if self.table and self.table == curFocus.table:
+		if self.table and self.table == curFocus.table and self.table.IAccessibleTable2Object.nSelectedCells > 0:

```

Comment:
Added newlines for readability

```suggestion
		if (
			self.table
			and self.table == curFocus.table
			and self.table.IAccessibleTable2Object.nSelectedCells > 0
		):
```